setup.py

```diff
@@ -18,10 +18,10 @@
     include_package_data=True,
     setup_requires=['setuptools-markdown'],
     install_requires=[
-        "eth-utils>=0.5.0",
-        "cytoolz>=0.8.2",
+        "eth-utils>=0.5.0<1.0.0",
```

Comment:
Interesting that this works without the comma, I thought it would fail.

Reply:
Unintential, that's a typo on my part.